Dockerfile

```
@@ -1,4 +1,4 @@
-FROM wordpress:latest
+FROM wordpress:5.5.1
```

```suggestion
FROM wordpress:5.5.1-php7.3
```
Is it possible to change this to PHP 7.3 for the time being to avoid any 7.4 related issues (deprecation and notice messages)